manifests/l/lencx/ChatGPT/0.7.4/lencx.ChatGPT.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: lencx.ChatGPT
+PackageVersion: 0.7.4
+PackageLocale: en-US
+Publisher: lencx
+PackageName: ChatGPT Desktop Application

```

Comment:
```suggestion
PackageName: ChatGPT
```
![image](https://user-images.githubusercontent.com/71691514/210417973-1d6e9330-517a-4af7-b707-acb77d922e20.png)
This should match control panel